src/features/stats/index.js

```diff
@@ -38,9 +39,33 @@ class Stats extends Component {
 
     render() {
         const { disabled, stats, sideMenu, largeView } = this.props;
-        const { abilities, level, exp, expToLevel, hp, maxHp, gold } = stats;
+        const {
+            abilities,
+            level,
+            exp,
+            expToLevel,
+            hp,
+            maxHp,
+            mana,
+            maxMana,
+            gold,
+            start,
+        } = stats;
         const { statsBgColor } = this.state;
 
+        let STRbonus = calculateModifier(abilities.strength);
+        let CONbonus = calculateModifier(abilities.constitution);
+        let DEXbonus = calculateModifier(abilities.dexterity);
+        let CHRbonus = calculateModifier(abilities.charisma);
+        let INTbonus = calculateModifier(abilities.intelligence);
+        let WISbonus = calculateModifier(abilities.wisdom);
+
+        stats.maxMana = calculateMaxManapool(INTbonus);
+        if (abilities.intelligence >= 10 && stats.start) {
+            stats.mana = stats.maxMana;
+            stats.start = false;
+        }
```

Comment:
Why are these `stats.maxMana` instead of simply `maxMana`? The assignment near the top of the file means you don't need to prepend with `stats` (unless there's a subtle difference I don't know about).

Reply:
Tested and pre-pending with stats isn't required. Changes have been made.